src/controller_examples/rabbitmq_controller/spec/reconciler.rs

```diff
@@ -778,7 +778,13 @@ pub open spec fn make_rabbitmq_pod_spec(rabbitmq: RabbitmqClusterView) -> PodSpe
             seq![
                 ContainerView::default()
                 .set_name(new_strlit("setup-container")@)
-                .set_image(new_strlit("rabbitmq:3.11.10-management")@)
+                .set_image(rabbitmq.spec.image)
+                .set_resources(
```

Comment:
This reminds me: we should have a `resources` field for rabbitmq, like zookeeper

Reply:
We already have one..

Reply:
just realize this is for the init_container, then nvm